src/components/chat/chat.tsx

```diff
@@ -140,6 +193,14 @@ export const Chat: FC<Props> = ({ roomId, room }) => {
     <Stack sx={styles.container(stage)}>
       <Stack component="section" sx={styles.section}>
         <HostChatPrompt stage={stage} message={hostMessageData?.message} />
+        {stage === "chat" && (
+          <Stack sx={styles.typingDialog(isTyping)}>
```

Comment:
I wonder if this should be an independent component, so that we can encapsulate all the logic involved in `whoIsTyping`. The chat component is starting to get a bit messy.